src/styles/index.scss

```
@@ -1,3 +1,115 @@
+@import '../styles/text_style';
+@import '../styles/stars_style';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.catalog {
+  &_img {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 32px auto 40px;
+  }
+
+  &_name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #{$text-color};
+    display: block;
+    margin: 0;
+    padding: 0 16px;
+    height: 36px;
+    width: 166px;
+  }
+
+  &_reviews {
+    font-family: Roboto, Arial, sans-serif;
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    width: 53px;
+    height: 16px;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    color: #{$text-color};
+    margin: 0;
+    padding-left: 13px;
+  }
+
+  &_code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #{$little-text-color};
+    width: 98px;
+    height: 14px;
+    margin: 4px 0 16px;
+    padding-left: 16px;
+  }
+
+  &_price-container {
+    display: flex;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &_price {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #{$little-text-color};
+    margin: 0;
+    padding-left: 16px;
+  }
+
+  &_price-value {
+    display: flex;
+    color: #{$text-color};
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    margin: 0 0 0 auto;
+    padding-right: 16px;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &_price {
    font-size: 12px;
    font-weight: 400;
    line-height: 18px;
    color: #{$little-text-color};
    margin: 0;
    padding-left: 16px;
    
    &-container {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }

```